package.json

```diff
@@ -105,7 +105,7 @@
 		"autosize": "4.0.2",
 		"autwh": "0.1.0",
 		"bcryptjs": "2.4.3",
-		"bootstrap": "^4.3.1",
+		"bootstrap": "4.3.1",
```

Comment:
This was here because your dependency on bootstrap pulls in `^4.3.1`, so I assumed it would make sense to have matching versions